task3.js

```diff
@@ -0,0 +1,66 @@
+const taskOneButton = document.getElementsByClassName('task-one-button')[0]
+const downloads = [
+  {
+    id: 1,
+    title: 'Recipe',
+    status: 'Done',
+  },
+  {
+    id: 2,
+    title: 'Workouts',
+    status: 'Pending',
+  },
+  {
+    id: 3,
+    title: 'Passwords',
+    status: 'Pending',
+  },
+  {
+    id: 4,
+    title: 'To Do 2021',
+    status: 'Pending',
+  },
+  {
+    id: 5,
+    title: 'Books',
+    status: 'Failed',
+  },
+]
+
+function changeItemProperty() {
+  const currentInterval = setInterval(main, 5000)
+  function main() {
+    console.log('Check started')
+    const filtered = downloads.filter((e) => e.status === 'Pending')
```

Comment:
Almost done, but you have an extra array iteration. You don't need to filter and then search for an element. you can just find an element `const nextItem = downloads.find((e) => e.status === 'Pending')` and then check for availability. If it is - change the status, if not - go ahead.

Reply:
It makes sence. Thank you for the notice. I fixed this part of code and I hope it's correct this time